Kickstarter-iOS/Views/Cells/PledgeShippingLocationCell.swift

```diff
@@ -1,8 +1,10 @@
+import KsApi
```

Comment:
Importing `KsApi` is only necessary to convert the `currencyCode` into a `Country`. This was only meant to be temporary because this PR was created before the shipping amount formatting work was finished. I see that's all merged now so I'll update this PR accordingly.